src/playermanager.h

```diff
@@ -106,6 +109,7 @@ class ZEPlayer
 private:
 	bool m_bAuthenticated;
 	bool m_bConnected;
+	const CSteamID* m_UnauthenticatedSteamID;
```

Comment:
Is there any reason why this is needed?

Reply:
To make it more obvious if we misuse an unauthenticated steam ID, thinking its real. Poggu suggested this and I agree with the reasoning https://discord.com/channels/1088140212602949644/1116781097473294477/1186034430364622978.